### PR TITLE
Fix typos in models/

### PR DESCRIPTION
**Pull request by ethantkoenig**
The title seems self-explanatory.

### CODE REVIEW

File: models/repo.go [hunk 1]
```diff
@@ -1756,7 +1756,8 @@ func DeleteMissingRepositories() error {
 	return nil
 }
 
-// ReinitMissingRepositories reinitializes all repository records that lost Git files.
+// ReinitMissingRepositories re
+// initializes all repository records that lost Git files.
```
Comment (strk) [inline models/repo.go]: Is "reinitializes" not a word ?

**Review by strk — APPROVED** [review]
Other than nitpicking, LGTM

**Comment by strk** [comment]
LGTM (said inside the review doesn't seem to be handled)

File: models/repo.go (hunk 1 above)
Comment (ethantkoenig) [reply models/repo.go]: Whoops this is a mistake, will fix

**Comment by Bwko** [comment]
LGTM

**Pull request merged**